yarn.lock

```diff
@@ -1821,9 +1821,9 @@ tsconfig@^6.0.0:
     strip-json-comments "^2.0.0"
 
 tslib@^1.7.1, tslib@^1.8.0, tslib@^1.8.1:
-  version "1.9.3"
-  resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.9.3.tgz#d7e4dd79245d85428c4d7e4822a79917954ca286"
-  integrity sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ==
+  version "1.10.0"
```

Comment:
we need to update `"tslib"` in the "dependencies" block as well, if we are testing against this version in development

Reply:
I'm not sure if I understand, the usage of caret in version number `^1.x.x`, it is already being resolved to latest version which is `1.10.0`.

<img width="434" alt="Screenshot 2019-10-01 at 21 02 56" src="https://user-images.githubusercontent.com/3933866/65992110-e0d42800-e48e-11e9-8c2e-40dce223bde0.png">

Reply:
Sometimes features are added in `typescript` which require the latest version of the `tslib` library at runtime. So it's better to be safe and have tslint bump its minimum dependency to be `"tslib": "^1.10.0"`

Reply:
Done in 176e920